docker-app/qfieldcloud/filestorage/view_helpers.py

```diff
@@ -233,6 +235,32 @@ def download_field_file(
     http_host = request.headers.get("host", "")
     https_port = http_host.split(":")[-1] if ":" in http_host else "443"
 
+    download_range = request.headers.get("Range", "")
+    if download_range:
+        range_match = re.match(r"bytes=(\d+)-(\d+)?", download_range)
+
+        if not range_match:
+            raise BadRequest(
+                "`Range` HTTP header must be formatted like `bytes=start-(end)`"
+            )
+
+        file_size = field_file.size
+        range_start = int(range_match.group(1))
+
+        if range_match.group(2):
+            range_end = int(range_match.group(2))
+        else:
+            range_end = file_size - 1
+
+        if (
+            range_start >= file_size
+            or range_end >= file_size
+            or range_end < range_start
+        ):
+            raise BadRequest("Invalid file range")
```

Comment:
```suggestion
            raise BadRequest("Invalid value for `Range` HTTP header`")
```
Just to be aligned with the previous message.